apps/src/music/MusicView.jsx

```diff
@@ -147,6 +148,32 @@ class UnconnectedMusicView extends React.Component {
     this.player.clearAllSoundEvents();
   };
 
+  uploadSound = file => {
```

Comment:
Thoughts on moving this into its own file? I just want to make sure, since this is temporary code, that we have a clear idea of which pieces are temporary/hack vs which we want to keep around. I'm thinking it could just be a `SoundUploader` class that takes in the `player` in its constructor and has one `uploadSound` function. It could maintain the `currentUploadSoundIndex` internally too

Reply:
Great instincts.  Done.